docs/source/whats_new.rst

```diff
@@ -8,6 +8,14 @@ What's New
 Current
 -------
 
+Bug
+~~~
+
+- In find_noisy_channels the signal detrend was accidentally undone which destabilized the RANSAC.
+- The detrend is now done over the NoisyChannels.raw_mne object to avoid this and to force that any signal in there is detrended like in matlab's prep.
```

Comment:
the two lines above are both relating to the same thing, so it'd be cool if you could make that a single list entry